pages/common/quarto.md

```diff
@@ -3,34 +3,22 @@
 > An open-source scientific and technical publishing system built on Pandoc.
 > More information: <https://quarto.org/>.
 
-- Create a Quarto project:
+- Create a new project:
 
-`quarto create-project {{path/to/project}}`
+`quarto create-project {{path/to/destination_directory}} --type {{book|default|website}}`
 
-- Create a new website project:
+- Create a new blog website:
 
-`quarto create-project {{path/to/destination_directory}} --type {{website}}`
+`quarto create-project {{path/to/destination_directory}} --type {{website}} --template {{blog}}`
 
-- Create a new book project:
+- Render input file(s) to different formats:
 
-`quarto create-project {{path/to/destination_directory}} --type {{book}}`
+`quarto render {{path/to/file.qmd|rmd|ipynb]}} --to {{html|pdf|docx}}`
```

Comment:
```suggestion
`quarto render {{path/to/file.qmd|rmd|ipynb}} --to {{html|pdf|docx}}`
```

Reply:
Thank you!

Reply:
What is the proper way to indicate multiple choices? I based my suggestion on the `tar` entry:

![image](https://user-images.githubusercontent.com/4519221/200561412-f92edca2-fcd7-48f0-b626-fb947d606067.png)

Reply:
I think it should read `quarto render {{path/to/file.qmd[.rmd|.ipynb]}} --to {{html|pdf|docx}}`

Reply:
We could use 
```suggestion
`quarto render {{path/to/file.{{qmd|rmd|ipynb}}}} --to {{html|pdf|docx}}`
```
But this wont change the appearance of how it renders, but your suggestion @PMassicotte doesn't match our syntax

Reply:
All good! Thank you for helping.